plugins/modules/k8s.py

```diff
@@ -87,6 +87,7 @@
       will only work if the same object is passed with state=absent (alternatively, just use state=absent with the name including
       the generated hash and append_hash=no)
     - Requires openshift >= 0.7.2
+    default: False
```

Comment:
I'm a bit confused too because I just saw these same changes in the PR #324.

Reply:
Yeah I didn't know about @Akasurde fix until @geerlingguy told me.